### PR TITLE
Update wolfi hashes

### PR DESCRIPTION
**Pull request by pjlast**
Ran `sg wolfi update-hashes`

## Test plan

Hashes updated.

### CODE REVIEW

**Comment by burmudar** [comment]
I ran `sg wolfi lock` and ... now there are a lot more updates 😬

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ repos:
     hooks:
       - id: check-yaml
         exclude: 'pnpm-lock.yaml'
-      - id: end-of-file-fixer
+      #      - id: end-of-file-fixer
```
Comment (burmudar) [inline .pre-commit-config.yaml:8]:
```suggestion
      - id: end-of-file-fixer
```
oops

File: .pre-commit-config.yaml
Comment (eseliger) [inline .pre-commit-config.yaml:1]: cough

**Review by eseliger — APPROVED** [review]
pending revert of the pre commit config change lgtm

**Comment by burmudar** [comment]
Looking into the build failure

File: dev/oci_deps.bzl [hunk 2]
```diff
@@ -35,7 +35,7 @@ def oci_deps():
     # Tag: docker:27.0.3-dind
```
Comment (willdollman) [inline dev/oci_deps.bzl:35]: @burmudar would you mind updating the tag you're pulling from

Comment (burmudar) [reply dev/oci_deps.bzl:35]: oh this is still the same tag just the latest image of that tag

**Pull request merged**